File: Corrections/java/Neufplate/src/main/java/fr/neufplate/PriceCalculator/ExcludingTaxesPriceDecorator.java
package fr.neufplate.PriceCalculator;

public class ExcludingTaxesPriceDecorator extends PriceDecorator {
    public ExcludingTaxesPriceDecorator(Price price) {
        super(price);
    }

    @Override
    public double calculate() {
        this.baseValue = super.calculate() * 1000.0d;
        return this.baseValue;
    }
}

File: Corrections/php/Neufplate/src/PriceCalculator/ExcludingTaxesPriceDecorator.php
<?php

namespace App\PriceCalculator;

class ExcludingTaxesPriceDecorator extends PriceDecorator
{


    public function __construct(Price $price)
    {
        parent::__construct($price);
    }

    public function calculate(): float
    {
        $this->baseValue = parent::calculate() * 10;
        return $this->baseValue;
    }

}